sonic_y_cable/y_cable_base.py

```diff
@@ -72,19 +70,33 @@ class YCableBase(Logger):
     PRBS_DIRECTION_GENERATOR = 1
     PRBS_DIRECTION_CHECKER = 2
 
-    def __init__(self, port, log_identifier):
+    def __init__(self, port, main_logger):
         """
         Args:
             port:
                  an Integer, the actual physical port connected to a Y cable
+            main_logger:
+                 a logging instance object, the main logging instance which can be used to log messages into syslog
+                 which can be called from derived classes using functions defined as log_info, log_warning etc.
```

Comment:
I think we can just call it `logger` instead of `main_logger`. The docstring should mention that it must be an instance of the Logger class from sonic_py_common instantiated by the program which is instantiating this class.

Reply:
fixed